frontend/src/metabase-types/store/visualizer.ts

```diff
@@ -51,30 +51,26 @@ export type VisualizerColumnValueSource =
   | VisualizerColumnReference
   | VisualizerDataSourceNameReference;
 
-export type VisualizerHistoryItem = {
+export type VisualizerVizState = {
   display: VisualizationDisplay | null;
   columns: DatasetColumn[];
   columnValuesMapping: Record<string, VisualizerColumnValueSource[]>;
   settings: VisualizationSettings;
 };
```

Comment:
After the refactoring, `VisualizerHistoryItem` didn't look like a good name, so I renamed it to `VisualizerVizState`